src/sentry/static/sentry/app/components/loadingError.tsx

```diff
@@ -1,9 +1,12 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import classNames from 'classnames';
 import {t} from 'app/locale';
 
-class LoadingError extends React.Component {
+type Props = {
+  onRetry?: () => void;
+  message?: string;
```

Comment:
I think we want the equivalent of `PropTypes.node` (idk what it is, cc @dashed), and it should probably be required

Reply:
hmm we def just have `return <LoadingError />;`  in places

Reply:
AH ok

Reply:
`React.ReactNode` is the type